services/crypto_get_info.proto

```diff
@@ -182,10 +182,22 @@ message CryptoGetInfoResponse {
          */
         StakingInfo staking_info = 22;
 
+        message VirtualAddressInfo {
+            /**
+             * A virtual address entry.
+             */
+            VirtualAddress virtual_address = 1;
+
+            /**
+             * The ethereum transaction nonce associated with this virtual address.
+             */
+            int64 nonce = 2;
```

Comment:
From an end user perspective, this is bit unwieldy. I think we should have a flattened structure returned by queries that has all the necessary info in one `message`. I understand for code reuse it makes sense to have the above but we have to design APIs for end users.

Reply:
@steven-sheehy what's the suggestion here to flatten it?

Reply:
```proto
message VirtualAddressInfo {
    /**
     * The 20-byte EVM address that is derived from the keccak-256 hash of a ECDSA_SECP256K1 primitive key.
     */
    bytes address = 1;

    /**
     * Flag if this address should now be set or is the default address on the account.
     */
    bool is_default = 2;

    /**
     * The ethereum transaction nonce associated with this virtual address.
     */
    int64 nonce = 3;
}

Reply:
Thanks @steven-sheehy, updated as suggested.